docs/content/developers/maintainers.md

```diff
@@ -48,6 +48,8 @@ There are cases like access to hosting provider integrations that have essential
 * **CircleCI**: Maintainers automatically have some access via their GitHub team membership, but should probably get more.
 * **developer.apple.com**: Add to the DDEV team there so certificates can be managed.
 * **hub.docker.com**: Add user to owners team in DDEV org.
+* **AUR**: Add user to Co-maintainers (account should be created first) for `ddev-bin` and `ddev-edge-bin`.
+* **Packagist**: Add user to Maintainers (login with GitHub) for `ddev/ddev-test-composer-create`.
```

Comment:
@rfay, please create an account on https://packagist.org/, so I can add you.

Reply:
Account created, `rfay`, thanks.

Reply:
Done https://packagist.org/packages/ddev/ddev-test-composer-create